pantheon-bundle/frontend/src/app/__snapshots__/searchBeta.test.tsx.snap

```diff
@@ -315,7 +320,7 @@ exports[`SearchBeta tests should render default Search component 1`] = `
                 className=""
               >
                 <SimpleListItem
-                  className=""
+                  className="filters-drawer__repo-list-item"
```

Comment:
This seems more like:
`repo-list__item`

No need to indicate it's part of the filters-drawer since there aren't two repo lists on the page. If I was worried about that I'd add `filters-drawer__repo-list__item`, but I generally try to avoid BEM classes with more than one `__`.